pos_show_clock/static/src/js/ChromeWidgets/Clock.js

```diff
@@ -0,0 +1,40 @@
+odoo.define("pos_show_clock.Clock", function (require) {
+    "use strict";
+
+    const PosComponent = require("point_of_sale.PosComponent");
+    const Registries = require("point_of_sale.Registries");
+    const {useState} = owl;
+
+    class Clock extends PosComponent {
+        setup() {
+            super.setup();
+            this.state = useState({current_time: ""});
+            this.mounted();
+        }
+        mounted() {
+            this.updateCurrentTime();
+        }
+        updateCurrentTime() {
+            var self = this;
+            const now = Date.now();
+            const next_update = 60050 - parseInt(moment(now).format("ss")) * 1000;
+            // 1 min = 60.000ms
+            // 60000 + 50 (less then 100ms rule)
+            // it's the time we can update the minute without the user noticing
+            // the difference when comparing it to another watch.
+            this.state.current_time = moment(now).format("HH:mm");
+            setTimeout(() => {
```

Comment:
I would consider setInverval rather than setTimeout.
It feels better for this use case.

I would also store the timer reference (`this.invervalReference = setInterval(...)`). You can do it when mounting the component (`mounted`), and then clear it when the component is unmounted to keep things clean (`clearInterval(this.intervalReference)`)